src/lib/selector.c

```diff
@@ -292,10 +294,19 @@ int ncselector_additem(ncselector* n, const struct ncselector_item* item){
     n->longdesc = cols;
   }
   ++n->itemcount;
+  int dimy, dimx;
+  ncselector_dim_yx(n->ncp->nc, n, &dimy, &dimx);
+  if(origdimx < dimx || origdimy < dimy){ // resize if too small
+    ncplane_resize(n->ncp, 0, 0, 0, 0, 0, 0, dimy, dimx);
```

Comment:
We need an error check here, though admittedly I'm not sure what ought be done if it fails.

I'm surprised I haven't made `ncplane_resize()` a no-op when incoming size is equal to original size, which would allow this to be done without the check or the stashing of original values (technically that ought be possible now, as well, but it would involve some work). I think I'm going to make that change. But this can stand.

Such a no-op case would also require that everything be preserved (i.e. origdimy, origdimx would need be provided as the third and fourth arguments, and trimmed if we were shrinking). So what you have here is probably for the best, no matter what.